ntbs-service/Pages/Notifications/Edit/Patient.cshtml

```diff
@@ -184,7 +184,7 @@
             <label asp-for="Patient.Address" nhs-label-type="Standard">
                 Address
             </label>
-            <span nhs-span-type="ErrorMessage" id="address-error" ref="errorField"></span>
+            <span nhs-span-type="ErrorMessage" id="address-error" ref="errorField" asp-validation-for="Patient.Address"></span>
```

Comment:
The first thing my testing spotted - the validation was missing here so the error didn't actually get shown after submitting the form :)